README.md

```diff
@@ -16,7 +16,7 @@ Nova Inline Relationship allows you to manage (add/edit/update/delete/reorder) a
 
 ## Requirements
 
-This package requires Laravel Nova 4.0.
+This package requires Laravel Nova 4.26.
```

Comment:
Is this specific version really a requirement?

Reply:
@luisdalmolin it should be ^4.26! btw, composer.json currently defines a specific version (4.2) and I added to 4.26, but I'm still trying to make it working without changing it, but no success for now. 

PR marked as draft